modules/task/backend.go

```diff
@@ -78,31 +74,14 @@ func (b NopBackend) IsRunning() bool {
 type inMemBackend struct {
 	bufQueue  chan []byte
 	isRunning bool
+	isStopped bool
```

Comment:
Sounds like a enum-type struct would be better, no?

Especially since `isRunning = true, isStopped = true` is an invalid state and two booleans give you four.